docs/pages/project/contributing/contributing-docs.md

```diff
@@ -142,38 +153,45 @@ Meshery documentation is made of these components:
   $ make site
   ```
 - If that gives an error run:
+
   ```bash
     $ bundle exec jekyll serve
   ```
 
   _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artifacts for production._
 
 ### Using Docker
+
 If you've Docker and `make` installed in your system, then you can serve the site locally
-  ```
-  $ make docker-docs
-  ```
+
+```
+$ make docker-docs
+```
 
 This doesn't require the need for installing Jekyll and Ruby in your system
 
-**But, you need to make sure that GNU make is working in your system (might not work in Windows)** 
+**But, you need to make sure that GNU make is working in your system (might not work in Windows)**
+
 #### Note
```

Comment:
Nice changes you made there for readability! GG!

Reply:
Yeah that's my mistake, I will fix it. Thanks a lot for pointing it out @alphaX86

Reply:
I have fixed the misalignment, please point out if there is any other issue with the PR @alphaX86